src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -457,24 +457,20 @@ public void StopReceiving()
         /// If you're having any trouble setting up webhooks, please check out this <see href="https://core.telegram.org/bots/webhooks">amazing guide to Webhooks</see>.
         /// </remarks>
         /// <see href="https://core.telegram.org/bots/api#setwebhook"/>
-        public Task SetWebhookAsync(string url = "", FileToSend? certificate = null,
-            int maxConnections = 40,
-            UpdateType[] allowedUpdates = null,
+        public async Task<bool> SetWebhookAsync(
+            string url = "",
```

Comment:
This tg API methods is a bit confusing. passing empty url here is exatcly the same as calling `deleteWebhook`!

We shouldn't have default value for `url` param. Users can either pass empty here or call `DeleteWebhookAsync` to remove the webhook.

Reply:
Also, if we do such as change, let's have it on the migration wiki.

Reply:
Agree, I don't like it. I'm for removing default url.